src/main/resources/assets/minecolonies/lang/en_us.json

```diff
@@ -1775,5 +1775,6 @@
   "com.minecolonies.coremod.gui.locate": "Locate",
   "com.minecolonies.coremod.locating": "Successfully located rack with respective item(s).",
 
-  "entity.researcher.moreknowledge": "During your absence, your Researchers used the time to do some extra work. This will speed up their current research a bit!"
+  "entity.researcher.moreknowledge": "During your absence, your Researchers used the time to do some extra work. This will speed up their current research a bit!",
+  "com.minecolonies.coremod.gui.request.hideshow": "Only show important requests"
```

Comment:
```suggestion
  "com.minecolonies.coremod.gui.request.hideshow": "Only show important requests:"
```

(Assuming there's an On/Off button after this)

Reply:
No, this is the tooltip